impeller/renderer/formats.h

```diff
@@ -11,6 +11,7 @@
 
 #include "flutter/fml/hash_combine.h"
 #include "flutter/fml/macros.h"
+#include "fml/logging.h"
```

Comment:
The world's smallest nit: "flutter/fml/logging.h"

Reply:
Roger. I think clangd gets confused when the same file can be reached via multiple `-I` directives. I manually correct these but forgot to address this one.